OpenCorePkg.dsc

```diff
@@ -318,12 +318,12 @@
   DEFINE OCPKG_BUILD_OPTIONS_GEN = -D DISABLE_NEW_DEPRECATED_INTERFACES $(OCPKG_BUILD_OPTIONS) -D OC_TARGET_$(TARGET)=1
   DEFINE OCPKG_ANAL_OPTIONS_GEN = "-DANALYZER_UNREACHABLE=__builtin_unreachable" "-DANALYZER_NORETURN=__attribute__((noreturn))"
 
```

Comment:
Add `gOpenCorePkgTokenSpaceGuid.PcdCanaryAllowRdtscFallback|TRUE` to `[PcdsFixedAtBuild]` in OpenCorePkg.dsc and OpenDuetPkg.dsc, since we need to support older systems.